src/storage_stage.rs

```diff
@@ -436,19 +437,26 @@ mod tests {
 
     #[test]
     fn test_pubkey_distribution() {
-        logger::setup();
         // See that pub keys have an even-ish distribution..
-        let mut hist = [0; NUM_IDENTITIES];
-        for _ in 0..(128 * 256) {
```

Comment:
cool!

how is 128*256 related to NUM_IDENTITIES?

Reply:
It's enough pub keys that hits all the histogram slots which is kind of a bare-minimum success condition for this test. Possibly I could use less and look at something else like variance of histogram values maybe. But this doesn't take too long now.

Reply:
NUM_IDENTITIES IS 1024, so it's 32 * NUM_IDENTITIES. I changed the code to match that.